artemis/modules/nuclei.py

```diff
@@ -104,7 +104,9 @@ def run_multiple(self, tasks: List[Task]) -> None:
             "-headless-bulk-size",
             str(len(tasks_filtered)),
             "-milliseconds-per-request",
-            str(int(Config.Limits.SECONDS_PER_REQUEST * 1000.0 / len(tasks_filtered))),
+            str(int((1 / Config.Limits.REQUEST_PER_SECOND) * 1000.0 / len(tasks_filtered)))
+            if Config.Limits.REQUEST_PER_SECOND != 0
+            else str(int(0)),
```

Comment:
why int(

Reply:
bump

Reply:
I changed the value here, not the structure. It was earlier here, and I assumed it had its purpose. But if you change the structure, it won't work (nuclei takes int), and tests will (obviously) not pass in one of our config conditions. So - do you propose something specific here, or you just asked?

Reply:
I meant that 'else str(int(0))' can be reduced to 'else str(0)'